[CI:DOCS] Remove 'you' from man pages (#16751)
containers/podman: docs/source/markdown/options/pod.run.md

####> are applicable to all of those.
#### **--pod**=*name*

> Run container in an existing pod. Podman can make the pod automatically by prefixing the pod name with **new:**.

Reviewer: ```suggestion
Run the container in an existing pod. Podman will make the pod automatically if the pod name is prefixed with **new:**.
```
